.github/workflows/commit.yml

```diff
@@ -0,0 +1,138 @@
+---
+name: commit
+
+on:
+  push:
+
+
+jobs:
+  test:
+    name: lint and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: build
+        run: make build
+      - name: Archive Catalog Docker image
+        run: docker save -o ckan/image-catalog.tar datagov/catalog.data.gov:latest
+      - name: Archive Solr Docker image
+        run: docker save -o ckan/image-solr.tar datagov/catalog.data.gov.solr:latest
+      - name: Archive DB Docker image
+        run: docker save -o ckan/image-db.tar datagov/catalog.data.gov.db:latest
+      - uses: actions/upload-artifact@v2
+        with:
+          name: docker-images
+          path: ckan/
+          if-no-files-found: error
```

Comment:
Do we need to archive these? Where is the docker push happening?

Reply:
It's not yet, but I was able to do that part quickly. Created an issue (see description of this PR) to backlog this work.